SparkPost/2web2ui
# FE-1353 - dont show dashboard onboarding for on prem (allow_events_ingest)

### What Changed
- Skip onboarding on the dashboard for on prem users.

### How To Test
- Load up an account that has allow_events_ingest and go to the dashboard. (shouldn't see onboarding)
- Load up an account without allow_events_ingest and go to the dashboard. (should see onboarding)

### To Do
